src/modules/assets/views/AddAsset.vue

```diff
@@ -0,0 +1,380 @@
+<script>
+import BaseAlert from "@/components/BaseAlert.vue";
+import axios from "axios";
+export default {
+  name: "InviteUser",
+  data() {
+    return {
+        asset:'',
+        type:'',
+        requested_date:null,
+        requested_amount:'',
+        status:'',
+        bought_date:null,
+    };
+  },
+  methods: {
+    
+    formData() {
+      return {
+        asset:this.asset,
+        type:this.type,
+        requested_amount:this.requested_amount,
+        status:this.status,
+        bought_date:this.bought_date,
+        requested_date:this.requested_date
+      };
+    },
+    async handleSubmit() {
+      
+      try {
+        const response = await axios.post("http://localhost:3000/my_assets_list",this.formData());
```

Comment:
After requesting the assets list, then what?

Reply:
we do nothing, just render the list in frontend

Reply:
we just send the data to backend, sent successfully message will be added by saugat